modules/nf-core/porechop/abi/main.nf

```diff
@@ -26,8 +26,21 @@ process PORECHOP_ABI {
         --input $reads \\
         --threads $task.cpus \\
         $args \\
-        --output ${prefix}.fastq.gz \\
-        > ${prefix}.log
+        --output ${prefix}.porechop_abi.fastq.gz \\
```

Comment:
```suggestion
        --output ${prefix}.fastq.gz \\
```
Can you change the prefix defining line to be:
`def prefix = task.ext.prefix ?: "${meta.id}.porechop_abi"`
Then the default is to rename it as such, but it can be overriden with prefix as normal.
(won't let me suggest on unchanged lines).